src/views/About.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+<div class="background">
+<!-- Bootstrap container für den Abstand und die mittige Platzierung !-->
+<b-container center>
+    <div>
+
+        <b-row>
```

Comment:
Wäre es vielleicht sinnvoll die Überschrift in eine eigene Komponenten auszulagern, damit man sie mehrfach wieder verwenden kann?

Reply:
Ja stimmt, ist richtig. mache ich.